demo/demo.html

```diff
@@ -3,11 +3,15 @@
   <head>
     <title>Demo of Logic and View Integration</title>
     <script src="../dist/components.js"></script>
-    <script src="https://cdn.groupbycloud.com/sfx-all-plugins-bundle.js"></script>
+    <script src='./temp/sfx-all-plugins-bundle.js'></script> 
+    <!-- above tag to be removed when the cdn PR passes with the updated bundle -->
+    <!-- <script src="https://cdn.groupbycloud.com/sfx-all-plugins-bundle.js"></script> -->
   </head>
   <body>
-    <sfx-search-box placeholder="search" clearbutton searchbutton></sfx-search-box>
-    <sfx-sayt visible></sfx-sayt>
+    <sfx-search-box placeholder="search" clearbutton searchbutton id='searchbox1'></sfx-search-box>
+    <sfx-sayt title='basic' searchbox='searchbox1'></sfx-sayt>
```

Comment:
Is there any value in using the title attribute for the two `sfx-sayt` components? Also, does `minch` have any value?